src/main/java/com/strandls/document/service/Impl/DocumentServiceImpl.java

```diff
@@ -165,7 +187,7 @@
 	private ActivitySerivceApi activityService;
 
 	@Inject
-	private TaxonomyServicesApi taxonomyService;
+	private SpeciesServicesApi speciesService;
```

Comment:
add this injected service to conetxt listerner before using it

Reply:
toxonomy service api Sprecies service need binding